estate/demo/estate_property.xml

```diff
@@ -0,0 +1,61 @@
+<odoo>
+    <record id="estate_property_1" model="estate.property">
+        <field name="name">Big Villa</field>
+        <field name="state">new</field>
+        <field name="description">A nice and big villa</field>
+        <field name="postcode">12345</field>
+        <field name="date_availability">2020-02-02</field>
```

Comment:
This should preferably be relative to the date of install (see codebase for examples)